C#/cs01/16.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Globalization;
+namespace Brainchild.Curriculum.Solutions
+{
+    public static class Pascal
+    {
+        public static string ToPascalCase(this string any)
+        {
+            TextInfo myTI = new CultureInfo("en-US", false).TextInfo;
+            return myTI.ToTitleCase(any);
```

Comment:
please do not use built in methods

Reply:
Ok,.shall i remove this program and rewrite with char[] logic?

Reply:
changed code.